docs/docs/en/guide/upgrade.md

```diff
@@ -1,59 +1,71 @@
+### Stop All Services of DolphinScheduler
 
-    ```shell
-    export DATABASE=${DATABASE:-mysql}
-    export SPRING_PROFILES_ACTIVE=${DATABASE}
-    export SPRING_DATASOURCE_DRIVER_CLASS_NAME=com.mysql.cj.jdbc.Driver
-    export SPRING_DATASOURCE_URL=jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
-    export SPRING_DATASOURCE_USERNAME={user}
-    export SPRING_DATASOURCE_PASSWORD={password}
-    ```
+Stop all services of dolphinscheduler according to your deployment method. If you deploy your dolphinscheduler according to [cluster deployment](./installation/cluster.md), you can stop all services by command `sh ./script/stop-all.sh`.
 
-- Execute database upgrade script:
+### Upgrade Database
 
-    `sh ./tools/bin/upgrade-schema.sh`
+Change configuration in `./bin/env/dolphinscheduler_env.sh` ({user} and {password} are changed to your database username and password), and then run the upgrade script.
 
-## Backend Service Upgrade
+Using MySQL as an example, change the value if you use other databases. Please manually download the [mysql-connector-java driver jar](https://downloads.MySQL.com/archives/c-j/)
+jar package and add it to the `./tools/libs` directory, then change `./bin/ env/dolphinscheduler_env.sh` file
 
-### Modify the Content in `conf/config/install_config.conf` File
+```shell
+export DATABASE=${DATABASE:-mysql}
+export SPRING_PROFILES_ACTIVE=${DATABASE}
+export SPRING_DATASOURCE_DRIVER_CLASS_NAME=com.mysql.cj.jdbc.Driver
+export SPRING_DATASOURCE_URL=jdbc:mysql://127.0.0.1:3306/dolphinscheduler?useUnicode=true&characterEncoding=UTF-8
+export SPRING_DATASOURCE_USERNAME={user}
+export SPRING_DATASOURCE_PASSWORD={password}
+```
+
+Execute database upgrade script: `sh ./tools/bin/upgrade-schema.sh`
+K
+### Upgrade Service
+
+#### Change Configuration `bin/env/install_config.conf`
+
+- If you deploy with Pseudo-Cluster deployment, change it according to [Pseudo-Cluster](./installation/pseudo-cluster.md) section "Modify Configuration".
+- If you deploy with Cluster deployment, change it according to [Cluster](./installation/cluster.md) section "Modify Configuration".
+
+And them run command `sh ./bin/start-all.sh` to start all services. 
 
-- Standalone Deployment please refer to the [Standalone-Deployment](./installation/standalone.md).
-- Cluster Deployment please refer to the [Cluster-Deployment](./installation/cluster.md).
+## Notice
 
-#### Masters Need Attentions
+### Differences of worker group (before or after version 1.3.1 of dolphinscheduler)
 
-Create worker group in 1.3.1 version has a different design: 
+The architecture of worker group is different between version before or after 1.3.1 
 
-- Before version 1.3.1 worker group can be created through UI interface.
+- Before version 1.3.1(include itself) worker group can be created through UI interface.
```

Comment:
@lenboo I find out in version 300, we can create a worker group in ou UI again, Am I right, and should we change this doc? which version do we support adding worker group in UI again?